FlashDevelop/MainForm.cs

```diff
@@ -487,6 +487,14 @@ public new String ProductName
         }
 
         /// <summary>
+        /// Gets the command prompt executable (custom or cmd.exe by default).
+        /// </summary>
+        public string CommandPromptExecutable
+        {
+            get { return File.Exists(Settings.CustomCommandPrompt) ? Settings.CustomCommandPrompt : "cmd.exe"; }
```

Comment:
If `cmd.exe` is the default value it will always fail so it seems wasteful.
Maybe default to an empty string and test that the string isn't empty first.